src/main/webapp/css/style.css

```diff
@@ -24,22 +24,27 @@ body, html {
   max-width: 580px;
 }
 
-.auth-button {
+.auth-button, .setting-button, .home-button {
   font: 400 15px Roboto, Arial, sans-serif;
   color: white;
   background: #4d90fe;
```

Comment:
idk if it's just me but the color of the settings and auth button look different next to each other...

Reply:
ah, fixed, it's not just you.